app/views/orders/confirmation.html.erb

```diff
@@ -0,0 +1,70 @@
+<% subtotal = 0 %>
+
+<h4>Order Placed: <%= @order_items.order.updated_at.strftime("%m/%d/%Y at %R") %></h4>
+<h4>Order Status: <%= @order_items.order.status %></h4>
+
+<table class="table">
+  <tr>
+    <th>Name</th>
+    <th>Price</th>
+    <th>Qty</th>
+    <th>Total</th>
+  </tr>
+  <% @order_items.each do |item| %>
+  <tr>
+    <% product = Product.find(item.product_id) %>
+    <td> <%= link_to  product.name, product_path(product) %> </td>
+    <td> <%= number_to_currency(product.price) %> </td>
+    <td>
+      <%= item.quantity %>
+    </td>
+    <td> <%= number_to_currency(product.price * item.quantity) %> </td>
+
+		<% subtotal += (product.price * item.quantity) %>
+  </tr>
+  <% end %>
+
+	<tr>
+		<td colspan="2">
+		</td>
+		<td>
+			Subtotal:
+		</td>
+		<td>
+			<%= number_to_currency(subtotal) %>
+		</td>
+	</tr>
+	<tr>
+		<td colspan="2">
+		</td>
+		<td>
+			Tax:
+		</td>
+		<td>
+			<% tax = subtotal * 0.095 %>
```

Comment:
The 0.095 is what we call a "magic number." This means it's a number that totally satisfies a requirement, but has no grounding in application logic; it's just a number appropriate for a specific situation. What if you have a client in a place that doesn't have sales tax? What if there's a state and a local sales tax? What if the sales tax is 7%?

There are no tax collection project requirements though, so I'm not gonna harp on it too much.